packages/terra-framework-docs/src/terra-dev-site/test/tabs/Tabs/ClosableTab.test.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import Tabs from 'terra-tabs';
+import TabContentTemplate from './TabContentTemplate';
+import styles from './TabsTemplate.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ClosableTab = () => {
+  const tabsExample = [];
+  const [tabs, setTabs] = useState(tabsExample);
+
+  tabsExample.push(
+    {
+      label: 'Doctors',
+      key: 'CloseLabelTab',
+      content: 'Doctor details',
+    },
+  );
+
+  tabsExample.push(
+    {
+      label: 'Pharmacy',
+      icon: <IconBriefcase />,
+      isIconOnly: true,
+      key: 'CloseIconTab',
+      content: 'Pharmacy details',
+    },
+  );
+
+  tabsExample.push(
+    {
+      label: 'Customized Tab',
+      customDisplay: <div className={cx('custom-display')}>Customized Tab</div>,
+      key: 'CloseCustomTab',
+      content: 'Custom display',
+    },
+  );
+
+  tabsExample.push(
+    {
+      label: 'Disabled Tab',
+      key: 'CloseDisabledTab',
+      icon: <IconSearch />,
+      isDisabled: true,
+    },
+  );
+
+  tabsExample.push(
+    {
+      label: 'List of In-Patient Details Department Wise',
+      key: 'CloselongLabel',
+      content: 'List for In-Patient Details Department Wise',
+    },
+  );
+
+  tabsExample.push(
+    {
+      label: 'Appointments',
+      key: 'CloselastTab',
+      content: 'Appointments',
+    },
+  );
+
+  const handleTabClose = (itemKey) => {
+    const tabsArray = [...tabs];
+    const indexToRemove = tabsArray.findIndex(tab => tab.key === itemKey);
+    if (indexToRemove !== -1) {
+      tabsArray.splice(indexToRemove, 1);
+      setTabs(tabsArray);
```

Comment:
Need to verify this test example as it does not removes tab on click as well as on delete key press !!

Reply:
Thanks it is fixed https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41